.env.development

```diff
@@ -14,9 +14,14 @@ USER_INFO_COOKIE_NAME='edx-user-info'
 INTEGRATION_WARNING_DISMISSED_COOKIE_NAME='integration-warning-dismissed'
 SEGMENT_KEY=''
 PORT=8734
+SITE_NAME='edX'
 ALGOLIA_APP_ID=''
 ALGOLIA_SEARCH_API_KEY=''
 ALGOLIA_INDEX_NAME=''
 FULLSTORY_ORG_ID=''
 USE_API_CACHE=''
-FEATURE_ENROLL_WITH_CODES='true'
+LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
+LOGO_TRADEMARK_URL=https://edx-cdn.org/v3/default/logo-trademark.svg
+LOGO_WHITE_URL=https://edx-cdn.org/v3/default/logo-white.svg
+FAVICON_URL=https://edx-cdn.org/v3/default/favicon.ico
+FEATURE_ENROLL_WITH_CODES=true
```

Comment:
This PR changes this back from what you changed it to in your last PR, did you mean to do that?

Reply:
Yeah, they are technically the same thing and I felt lazy to change it lol; everything that comes through .env files are strings anyways.